src/components/TradeAssetSearch/components/CustomAssetAcknowledgement.tsx

```diff
@@ -75,15 +77,35 @@ export const CustomAssetAcknowledgement: React.FC<CustomAssetAcknowledgementProp
     // Add asset to the store
     dispatch(assetsSlice.actions.upsertAsset(asset))
 
-    // Add market data to the store
-    dispatch(
-      marketDataSlice.actions.setCryptoMarketData({
-        [asset.assetId]: { price: '0', marketCap: '0', volume: '0', changePercent24Hr: 0 },
-      }),
-    )
+    const emptyMarketData = {
```

Comment:
can use `defaultMarketData` defined in the `marketDataSlice`

Reply:
honestly probably don't even need to worry about setting it to default since the selectors default to the default if not set either

Reply:
That default object is great, thank you. I did set it below though, as if I don't the key is missing from the market data store, which prevents it from being a searchable asset on the assets page.

https://github.com/shapeshift/web/pull/7370/commits/9f7c5f1860557b4f526baab4c81aa1cdd6029341